app/src/main/res/xml/network_security_config.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <network-security-config>
-    <debug-overrides>
+    <debug-overrides cleartextTrafficPermitted="true">
```

Comment:
is this ok?

Reply:
no
thx for checking! =)

Reply:
but... maybe it is ok
for debug
but is is not related to this PR anyway